WordPressKit/WordPressComOAuthClient.swift

```diff
@@ -24,15 +24,18 @@ public final class WordPressComOAuthClient: NSObject {
     @objc public static let WordPressComOAuthErrorNewNonceKey = "WordPressComOAuthErrorNewNonceKey"
     @objc public static let WordPressComOAuthErrorDomain = "WordPressComOAuthError"
 
+    @objc public static let WordPressComOAuthDefaultBaseUrl = "https://wordpress.com"
+    @objc public static let WordPressComOAuthDefaultApiBaseUrl = "https://public-api.wordpress.com"
+
     enum WordPressComURL: String {
-        case oAuthBase = "https://public-api.wordpress.com/oauth2/token"
-        case socialLogin = "https://wordpress.com/wp-login.php?action=social-login-endpoint&version=1.0"
-        case socialLogin2FA = "https://wordpress.com/wp-login.php?action=two-step-authentication-endpoint&version=1.0"
-        case socialLoginNewSMS2FA = "https://wordpress.com/wp-login.php?action=send-sms-code-endpoint"
-        case oAuthRedirect = "https://wordpress.com/"
-
-        var url: URL {
-            return URL(string: self.rawValue)!
+        case oAuthBase = "/oauth2/token"
+        case socialLogin = "/wp-login.php?action=social-login-endpoint&version=1.0"
+        case socialLogin2FA = "/wp-login.php?action=two-step-authentication-endpoint&version=1.0"
+        case socialLoginNewSMS2FA = "/wp-login.php?action=send-sms-code-endpoint"
```

Comment:
Just neat picking here but can we remove the "New"? and just have socialLoginSMS2FA? It made me wonder if there is something like socialLoginOldSMS2FA.

Reply:
That makes more sense to me too but I'm lacking the context on why it is this way so I'm hesitant to change it. There is also a `socialNewSMS2FASessionManager`. Perhaps this is something that should be refactored in another change?